docs/scenarios/minikube/minikube.md

```diff
@@ -0,0 +1,99 @@
+# Minikube Extension Testing
+
+## Prerequisites
+
+Ensure the following are installed:
+
+- **Podman** or **Docker**
+- **Virtualization Software** for Windows/Mac (e.g., WSL, Hyper-V, Libkrun, AppleHV), with minimum requirements:
+  - **CPU**: 2 cores
+  - **Memory**: 2048 MB
```

Comment:
I do no think minikube will run with only 2 GB of memory properly. Is there some minimal requirements for minikube?

Reply:
Yes: 
- **Virtualization Software** for Windows/Mac (e.g., WSL, Hyper-V, Libkrun, AppleHV), **with minimum requirements**:
  - **CPU**: 2 cores
  - **Memory**: 2048 MB

Reply:
oh, nice. Ok then.